bson/raw_bson.py

```diff
@@ -13,6 +13,59 @@
 # limitations under the License.
 
 """Tools for representing raw BSON documents.
+
+Inserting and Retrieving RawBSONDocuments
+=========================================
+
+Example: Moving a document between different databases/collections
+
+.. testsetup::
+  from pymongo import MongoClient
+  client = MongoClient("localhost", 27017, document_class=RawBSONDocument)
+  client.drop_database('db')
+  client.drop_database('replica_db')
+
+.. doctest::
+
+  >>> import bson
+  >>> from pymongo import MongoClient
+  >>> from bson.raw_bson import RawBSONDocument
+  >>> client = MongoClient("localhost", 27017, document_class=RawBSONDocument)
+  >>> db = client.db
+  >>> result = db.test.insert_many([
+  ...  {'a': 1},
```

Comment:
This indentation seems a bit weird to me... Can we align the documents the same way as https://pymongo.readthedocs.io/en/stable/examples/aggregation.html